src/EnergyPlus/SizingManager.cc

```diff
@@ -1857,170 +1852,171 @@ void ManageSystemVentilationAdjustments(EnergyPlusData &state)
+                    if (thisTermUnitFinalZoneSizing.VpzMinByZoneSPSized) {
                         OutputReportPredefined::PreDefTableEntry(
-                            state, state.dataOutRptPredefined->pdchS62zhdVpzminSPSize, TermUnitFinalZoneSizing(termUnitSizingIndex).ZoneName, "Yes");
+                            state, state.dataOutRptPredefined->pdchS62zhdVpzminSPSize, thisTermUnitFinalZoneSizing.ZoneName, "Yes");
                     } else {
                         OutputReportPredefined::PreDefTableEntry(
-                            state, state.dataOutRptPredefined->pdchS62zhdVpzminSPSize, TermUnitFinalZoneSizing(termUnitSizingIndex).ZoneName, "No");
+                            state, state.dataOutRptPredefined->pdchS62zhdVpzminSPSize, thisTermUnitFinalZoneSizing.ZoneName, "No");
                     }
                     Real64 VozHtg = 0.0;
-                    if (TermUnitFinalZoneSizing(termUnitSizingIndex).ZoneADEffHeating != 0.0) {
-                        VozHtg = VbzByZone(termUnitSizingIndex) / TermUnitFinalZoneSizing(termUnitSizingIndex).ZoneADEffHeating;
+                    if (thisTermUnitFinalZoneSizing.ZoneADEffHeating != 0.0) {
+                        VozHtg = VbzByZone(termUnitSizingIndex) / thisTermUnitFinalZoneSizing.ZoneADEffHeating;
                         OutputReportPredefined::PreDefTableEntry(state,
                                                                  state.dataOutRptPredefined->pdchS62zhdVozhtg,
-                                                                 TermUnitFinalZoneSizing(termUnitSizingIndex).ZoneName,
+                                                                 thisTermUnitFinalZoneSizing.ZoneName,
                                                                  VozHtg,
                                                                  4); // Voz-htg
                     }
                     // Outdoor Air Details Report - Design Zone Outdoor Airflow - Voz
                     Real64 VozMax = std::max(VozHtg, VozClg); // take larger of the heating and cooling Voz values
                     OutputReportPredefined::PreDefTableEntry(
-                        state, state.dataOutRptPredefined->pdchOaMvDesZnOa, TermUnitFinalZoneSizing(termUnitSizingIndex).ZoneName, VozMax, 4);
-                    state.dataOutRptPredefined->TotalVozMax +=
-                        VozMax * state.dataHeatBal->Zone(zoneNum).Multiplier * state.dataHeatBal->Zone(zoneNum).ListMultiplier;
+                        state, state.dataOutRptPredefined->pdchOaMvDesZnOa, thisTermUnitFinalZoneSizing.ZoneName, VozMax, 4);
+                    state.dataOutRptPredefined->TotalVozMax += VozMax *
+                                                               state.dataHeatBal->Zone(thisTermUnitFinalZoneSizing.ActualZoneNum).Multiplier *
+                                                               state.dataHeatBal->Zone(thisTermUnitFinalZoneSizing.ActualZoneNum).ListMultiplier;
```

Comment:
Use the correct zone number here for multipliers.

Reply:
agreed